packages/gatsby-source-wordpress/src/steps/create-schema-customization/build-types.js

```diff
@@ -72,34 +70,20 @@ const interfaceType = typeBuilderApi => {
     extensions: { infer: false },
   }
 
-  // if this is a node interface type
-  if (nodeInterfaceTypes.includes(type.name)) {
-    // we add nodeType (post type) to all nodes as they're fetched
-    // so we can add them to node interfaces as well in order to filter
-    // by a couple different content types
-    typeDef.fields[`nodeType`] = `String`
-    typeDef.interfaces = [`Node`]
```

Comment:
This is now added below 👇 in a different file as more types need to implement Node now